priv/repo/migrations/20241022133945_add_cluster_details_enrichment.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddClusterDetailsEnrichment do
+  use Ecto.Migration
+
+  def change do
+    alter table(:clusters_enrichment_data) do
+      add :details, :map
```

Comment:
I wonder what are the arguments for/against setting the default value at the schema vs at the migration level ?

Reply:
No arguments for/against it. Added also in the migration so we're 100% sure f8627b7b4ee0cde8cca8d95e33db9189254120ed